apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -119,7 +119,7 @@ export class SendMessageEmail extends SendMessageBase {
       return;
     }
 
-    const overrides = command.overrides[integration?.providerId] || {};
+    const overrides: Record<string, any> = command.overrides[integration?.providerId] || {};
```

Comment:
I think we need to change this, since line 224 will never execute because of this.

Reply:
Not sure I follow...